gear-test/src/sample.rs

```diff
@@ -88,6 +88,8 @@ pub struct Expectation {
     /// Flag, which points that errors are allowed. Could be used to check traps.
     #[serde(rename = "allowError")]
     pub allow_error: Option<bool>,
+    /// Expected active programs (not failed in the init) ids
+    pub active_programs: Option<Vec<ChainAddress>>,
```

Comment:
Need to provide "terminated_programs" also to separate progs from users

Reply:
Actual programs we get from `CollectState`, which now return only active ones. Consider doing in the next PR with `common::Program` and `common::Message` with core ones, because will be much easier and require less code to review.

Reply:
Same here. Provide issue and mention here, please

Reply:
#661